libraries/AP_Motors/AP_MotorsMatrix.cpp

```diff
@@ -560,6 +560,18 @@ void AP_MotorsMatrix::remove_motor(int8_t motor_num)
     }
 }
 
+void AP_MotorsMatrix::add_motors(const struct MotorDefInt *motors, uint8_t num_motors)
```

Comment:
comments above this method as well would be nice.

Reply:
Added